example/empty_queue.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Remove ingest queue entries for a given database 
+
+# Database names:
+# - cosmoDC2_v1_1_4_image_overlap
+# - dc2_run2_1i_dr1b
+# - dc2_errors
+# - dc2_object_run22i_dr6_wfd_v2_00
+
+# Default password
+# PASSWORD=CHANGEME
```

Comment:
Do you need all of these comments here (especially password)?

Reply:
:+1: